### PR TITLE
style(OMA): Fix indent

### PR DESCRIPTION
**Pull request by austin-schaefer**
Small indent fix

### CODE REVIEW

**Review by barbnewrelic — CHANGES_REQUESTED** [review]
If you look at this page in preview, your paragraph below the bullets is still aligning with the bullets. It should be aligned to left margin. Please fix, then go ahead and merge. No need for a second review from me. Thanks!

**Review by zuluecho9 — APPROVED** [review]

**Pull request merged**
